--- libbaseencode/CMakeLists.txt	2018-03-04 20:27:08.640596760 +0300
+++ CMakeLists.txt	2018-03-04 22:49:57.729947063 +0300
@@ -17,6 +17,9 @@
 
 set(CMAKE_C_FLAGS "-Wall -Werror -fPIC")
 
+include(GNUInstallDirs)
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}") # "lib" or "lib64"
+
 add_library(${PROJECT_NAME} SHARED ${SOURCE_FILES})
 
 target_link_libraries(${PROJECT_NAME} ${PROJECT_LIBS})
@@ -25,8 +28,8 @@
 
 install(
   TARGETS ${PROJECT_NAME}
-  ARCHIVE DESTINATION lib
-  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
+  LIBRARY DESTINATION ${LIB_INSTALL_DIR}
   COMPONENT library
 )
 
